src/main/resources/application.yml

```diff
@@ -21,6 +21,11 @@ spring:
             client-secret: ${system.client.secret:clientsecret}
             authorization-grant-type: client_credentials
             scope: read
+          offender-search-api:
+            provider: hmpps-auth
+            client-id: ${system.client.id:make-recall-decision-api}
+            client-secret: ${system.client.secret:clientsecret}
+            authorization-grant-type: client_credentials
         provider:
           hmpps-auth:
             token-uri: ${hmpps.auth.url:http://localhost:9090/auth}/oauth/token
```

Comment:
Should this point to dev for now? Or shall I keep it the same and just point to dev in the dev profile/